apps/static/src/theme.js

```diff
@@ -0,0 +1,9 @@
+(function () {
```

Comment:
question (blocking): I might be missing something here but why cant the useThemeSwitcher hook do this?

Reply:
Theme switcher works with delay. In my browser I can notice that theme is switched after header is rendered.

Reply:
Oh yeah I see it. Wondering if in future there is a way we can inline the script to apply the initial class rather than an external script that needs to be fetched.

Reply:
Sure we can inline it. but since we got couple applications I thinks it's just easier to add it as external script loaded async than trying to maintain it in every application separately.
Second thing is that we want to have theme class to appear asap, to avoid any flickering (ie, change body background color). Loading application takes some time, and there is long "no theme applied period" currently.